tests/countries/test_canada.py

```diff
@@ -491,7 +554,7 @@ def test_civic_holiday_ab(self):
 
```

Comment:
```python
        self.assertHolidayName(name, ab_opt_holidays, range(1974, 2050))
        self.assertNoHolidayName(name, ab_opt_holidays, range(1867, 1974))
```
If we already have full range subdiv_optional_holidays, we need to use it in full 😋